### PR TITLE
refactor data-studio's bottom job exec history

### PR DESCRIPTION
**Pull request by Zzm0809**
## Brief change log

image

image

image

image

image

image

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by gaoyan1998** [comment]
LGTM

File: dinky-core/src/main/java/org/dinky/job/JobConfig.java [hunk 1]
```diff
@@ -90,6 +90,7 @@ public class JobConfig {
             notes = "JSON configuration")
     private Map<String, String> configJson;
 
+    // todo: 需要删除, 且需要删除调用代码, 并需要确定是否重新处理删除后的判断逻辑
     @ApiModelProperty(
             value = "Flag indicating whether to use the result",
             dataType = "boolean",
```
Comment (aiwenmo) [inline dinky-core/src/main/java/org/dinky/job/JobConfig.java]: The front-end does not require this parameter, the back-end requires it.

**Review by aiwenmo — CHANGES_REQUESTED** [review]

**Comment by Zzm0809** [comment]
@aiwenmo PTAL

**Comment by Zzm0809** [comment]
@aiwenmo PTAL

**Review by aiwenmo — APPROVED** [review]
LGTM

**Pull request merged**